httpserver/service_node.cpp

```diff
@@ -331,18 +335,28 @@ void ServiceNode::bootstrap_swarms(
 
         if (relevant || swarms.empty()) {
 
-            /// what if not found?
-            size_t idx = swarm_id_to_idx[swarm_id];
+            to_relay[swarm_id].emplace_back(
+                entry.pubKey.c_str(), entry.bytes.c_str(), entry.hash.c_str(),
+                entry.ttl, entry.timestamp, entry.nonce.c_str());
```

Comment:
just wondering why you pass the c_str() to the `message_t` constructor?

Reply:
As in why I'm not using `std::string` in the constructor signature? I guess there is no reason now, but this way allows passing both `std::string` and `const char*` with minimal copies/constructions. It is considered a best practice in Rust :) but there is it more ergonomic. I will change to using `std::string` since I'm not using string literals anywhere.

Reply:
fixed